README.md

```diff
@@ -29,3 +29,50 @@ The package includes:
 
  * `WithCorrelationId()` - adds a `CorrelationId` to track logs for the current web request.
  * `WithCorrelationIdHeader(headerKey)` - adds a `CorrelationId` extracted from the current request header (or created if one does not exist).
+
+## Installing into an ASP.NET Core Web Application
+
+This is what your `Startup` class should contain in order for this enricher to work as expected:
+
+```cs
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog;
+
+namespace MyWebApp
+{
+    public class Startup
+    {
+        public Startup()
+        {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Debug()
+                .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {CorrelationId} {Level:u3}] {Message:lj}{NewLine}{Exception}")
+                .Enrich.WithCorrelationId()
+                .CreateLogger();
+        }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // ...
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
```

Comment:
default internal function do the same
services.AddHttpContextAccessor()

Reply:
Thanks! Sorted :+1: